src/notebooks/export/fileConverter.ts

```diff
@@ -18,6 +18,9 @@ import { ExportFileOpener } from './exportFileOpener';
 import { ExportUtilBase } from './exportUtil';
 import { ExportFormat, IExport, IExportDialog, IFileConverter, INbConvertExport } from './types';
 
+/**
+ * Converts different file formats to others. Used in export.
```

Comment:
Aren't some these documentations poinless?

Reply:
I believe that would be subjective.

Reply:
More constructive feedback here would be what would you rather have this say.